src/index.ts

```diff
@@ -109,14 +112,52 @@ export const bootstrapEmpackPackedEnvironment = async (
         });
 
         sharedLibsMap[pkg.name] = getSharedLibs(extractedPackage, '');
-
+        paths[pkg.filename] = formPackagesPaths(extractedPackage, '');
         saveFilesIntoEmscriptenFS(Module.FS, extractedPackage, '');
       })
     );
     await waitRunDependencies(Module);
   }
 
-  return sharedLibsMap;
+  return { sharedLibs: sharedLibsMap, paths: paths };
+};
+
+export interface IRemovePackagesFromEnvOptions {
+  /**
+   * The list of packages which should be removed
+   */
+  removeList: any;
+
+  /**
+   * The Emscripten Module
+   */
+  Module: any;
+
+  paths: { [key: string]: string };
+
+  /**
+   * The logger to use during the bootstrap.
+   */
+  logger?: ILogger;
+}
+
+/**
+ * Removing previously installed files
+ *
+ * @param options
+ * @returns void
+ */
+export const removingFiles = async (
+  options: IRemovePackagesFromEnvOptions
+): Promise<void> => {
+  const { removeList, Module, paths, logger } = options;
+  if (removeList.length) {
+    logger?.log('Removing files of previous installed packages');
+    removeList.map((pkg: any) => {
+      const packages = paths[pkg.filename];
+      removeFilesFromEmscriptenFS(Module.FS, packages);
+    });
```

Comment:
Small suggestion to make the logs more informative

```suggestion
    removeList.map((pkg: any) => {
      logger?.log(`Uninstalling ${pkg.name} ${pkg.version}`);
      const packages = paths[pkg.filename];
      removeFilesFromEmscriptenFS(Module.FS, packages);
    });
```